validator/client/validator.go

```diff
@@ -167,7 +166,6 @@ func (v *validator) WaitForSynced(ctx context.Context) error {
 			return errors.Wrap(err, "could not receive Synced from stream")
 		}
 		v.genesisTime = syncedRes.GenesisTime
-		break
```

Comment:
Same thing here @0xKiwi
https://github.com/prysmaticlabs/prysm/commit/25102e09781c35f5df887e9dcfc7c97b8c778edb

Reply:
This was needed, see above comment.